bleak/backends/winrt/client.py

```diff
@@ -756,14 +760,22 @@ def bleak_callback(s, d):
         characteristic_obj = characteristic.obj
         if (
             characteristic_obj.characteristic_properties
-            & GattCharacteristicProperties.INDICATE
+            & GattCharacteristicProperties.NOTIFY
         ):
-            cccd = GattClientCharacteristicConfigurationDescriptorValue.INDICATE
+            if kwargs.get("force_indicate", False) and (
+                characteristic_obj.characteristic_properties
+                & GattCharacteristicProperties.INDICATE
+            ):
+                # If we want to force indicate even when notify is available, also check if the device
+                # actually supports indicate as well.
+                cccd = GattClientCharacteristicConfigurationDescriptorValue.INDICATE
+            else:
+                cccd = GattClientCharacteristicConfigurationDescriptorValue.NOTIFY
         elif (
             characteristic_obj.characteristic_properties
-            & GattCharacteristicProperties.NOTIFY
+            & GattCharacteristicProperties.INDICATE
         ):
-            cccd = GattClientCharacteristicConfigurationDescriptorValue.NOTIFY
+            cccd = GattClientCharacteristicConfigurationDescriptorValue.INDICATE
         else:
             cccd = GattClientCharacteristicConfigurationDescriptorValue.NONE
```

Comment:
Not related to this PR, but `NONE` would disable notifications/indications. Should we raise an error here instead?